packages/lesswrong/components/search/SearchPageTabbed.tsx

```diff
@@ -369,6 +370,7 @@ const SearchPageTabbed = ({classes}: {
   const HitComponent = hitComponents[tab]
 
   return <div className={classes.root}>
+    <HeadTags title="Search" canonicalUrl='/search' />
```

Comment:
Why is "Search" needed here, doesn't it already have that title?

Reply:
Oh, right - I thought it was a required prop but it's not